Yoast/wpseo-woocommerce
# fixed CS and added needed grunt commands to create a artifact

## Summary

This PR can be summarized in the following changelog entry:

* Adds `copy:artifact` and `compress:artifact` command to Grunt to create an artifact zip.

## Relevant technical choices:

* using the grunt compress package
* using the grunt copy package
* adding a command that builds the project and creates a zipped artifact using the above named packages

## Test instructions

This PR can be tested by following these steps:

1. Ensure you have all development dependency's for the project by running `yarn install`
2.  Execute the new command `create:artifact`, this is an alias that contains the following commands: 
    * `update-version`
    *  `build`
    * `copy:artifact` 
    * `compress:artifact`
3. check if the command does what is described below
`create:artifact: Create a zipped artifact from a fresh build`
`copy:artifact: Copy the needed files to the ./artifact folder`
`compress:artifact: Create a compressed zip from the ./artifact folder`

Fixes #194
